checkov/common/bridgecrew/platform_integration.py

```diff
@@ -257,8 +257,10 @@ def setup_bridgecrew_credentials(
                     region_name=region,
                     config=config,
                 )
-                self.platform_integration_configured = True
-                self.use_s3_integration = True
+                if not is_package_scan:
+                    self.skip_fixes = True  # no need to run fixes on CI integration
+                    self.platform_integration_configured = True
+                    self.use_s3_integration = True
```

Comment:
@ayajbara Why would those 2 be false if this is a package scan?

Reply:
the name of the var maybe is not a best choice. 
but it used when we run package scanning and we didnt setup the bridgecrew credentials yet, since we need this setup for the new package scanning, I setup the credentials in the runner and using this var. 
and in this case these variables will be False since we need to block "upload results"